apps/web/src/common/modules/modals/RecommendedBrowserModal.vue

```diff
@@ -16,7 +16,7 @@
                    target="_blank"
                 >
                     <img class="browser-img"
-                         :src="`src/assets/images/${browser.img}`"
+                         :src="browser.img"
```

Comment:
question: why don't we use `srcset`?

Reply:
Oh I didn't know this. Thanx for information! 👍